### ISSUE TITLE
Mark 1.21.1 As Compatible...

### ISSUE THREAD

**Issue by Fyoncle** [Resourceful-Bees/Highlight#12]

**Comment by Mortycio** [comment]
Yeah, I also suggest adding the 1.21.1 tag to make future updates easier to sync for that specific version. Currently, it isn’t detected in the launcher's updates (Prism, in this case) since it was 'forcibly' downloaded by changing the filters to install it in a 1.21.1 instance.

**Comment by Foozey** [comment]
Agreed, please mark 1.21.1 as compatible!

**Comment by HatTrkPatrk** [comment]
Please.
